nexus/tests/output/nexus_tags.txt

```diff
@@ -1,14 +1,16 @@
 API operations found with tag "disks"
 OPERATION ID                             URL PATH
-project_disks_delete_disk                /organizations/{organization_name}/projects/{project_name}/disks/{disk_name}
-project_disks_get                        /organizations/{organization_name}/projects/{project_name}/disks
-project_disks_get_disk                   /organizations/{organization_name}/projects/{project_name}/disks/{disk_name}
-project_disks_post                       /organizations/{organization_name}/projects/{project_name}/disks
+disk_create                              /organizations/{organization_name}/projects/{project_name}/disks
+disk_delete                              /organizations/{organization_name}/projects/{project_name}/disks/{disk_name}
+disk_list                                /organizations/{organization_name}/projects/{project_name}/disks
+disk_view                                /organizations/{organization_name}/projects/{project_name}/disks/{disk_name}
```

Comment:
Not that it necessarily matters one way or the other, but GitHub's CLI uses `view` and `edit`. Regardless, it's all an upgrade to what we have today, ha.

Reply:
@karencfv we had agreed on `edit` -> `update` but were 50-50 on `view` vs. `get`. I guess I slightly prefer `get` since we might as well match the http method when it's natural to do so.

Reply:
I actually took the deliberate step **not** to match the HTTP method because I thought it might be confusing if some matched while others did not. That said: not a strong preference so if someone wants to make a case for one vs. the other that would be clarifying.

Reply:
I can definitely see that side too. It makes you assume things about the semantics that we may not want to imply. Or maybe we actually do because we want these to be spec compliant gets.

Reply:
I'm on the fence about this one as well 😅, but I think I'm leaning more towards the use of `get`. It's a tad more consistent with what people will be used to seeing in SDKs generally. That said, while `get` appears everywhere, there are also weird inconsistencies everywhere. Like for example, AWS uses some weird combination of ["sometimes `get`, sometimes `describe`"](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Operations.html). Both [Azure](https://docs.microsoft.com/en-us/rest/api/compute/disks) and [GCP](https://cloud.google.com/compute/docs/reference/rest/v1) have the more consistent APIs/SDKs and [VMware](https://developer.vmware.com/apis/vmc/latest/operation-index/) is all over the place. So , I guess sticking to `view` isn't a huge deal as long as we don't deviate.

TLDR: I have slight preference for `get` but not a huge deal if we stick with `view` :)

Reply:
I'm going to go with `_view`. I think of all the things to change in the future, `s/_view/_get/` is going to be pretty easy in API/SDK/CLI/etc.